types/react-rte/index.d.ts

```diff
@@ -148,6 +148,7 @@ interface Props {
     rootStyle?: object | undefined;
     editorStyle?: object | undefined;
     toolbarStyle?: object | undefined;
+    onBlur?: (event: Object) => void;
```

Comment:
You'll want a more specific type for the `event` argument here.  I have to check the `react-rte` package to verify what the target element is.  On a cursory glance it seems like a `div`?  So your type would be:
```
onBlur?: (event: React.FocusEvent<HTMLDivElement>) => void;
```
or
```
onBlur?: React.FocusEventHandler<HTMLDivElement>;
```